cuda_core/tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021-2024 NVIDIA Corporation.  All rights reserved.

```

Comment:
Nit: Lets have the year only be 2024. It's a new file for a new project, so dating it back to 2021 doesn't feel too right for me. Lets apply it to other files in this PR.